src/nisarqa/utils/stats_h5_writer/setup_writer.py

```diff
@@ -97,6 +106,83 @@ def add_global_metadata_to_stats_h5(
         )
 
 
+def setup_stats_h5_non_insar_products(
+    product: nisarqa.NonInsarProduct,
+    stats_h5: h5py.File,
+    root_params: nisarqa.typing.RootParamGroupT,
+) -> None:
+    """
+    Setup the STATS.h5 file for Non-InSAR products (RSLC, GSLC, GCOV).
+
+    Parameters
+    ----------
+    product : nisarqa.NonInsarProduct
+        Instance of a NonInsarProduct (RSLC, GSLC, GCOV).
+    stats_h5 : h5py.File
+        Handle to an HDF5 file where the identification metadata
+        should be saved.
```

Comment:
What is the "identification metadata" here? It seems like this function is now doing a bunch more besides saving identification metadata.

At some point recently, we updated the docstring descriptions for `product` and `stats_h5` in `_setup_stats_h5_all_products()`:

https://github.com/isce-framework/nisarqa/blob/75673c2257f1a1d343041684370459ee1e800f33/src/nisarqa/utils/stats_h5_writer/setup_writer.py#L21-L24

Let's make a similar update here.

```suggestion
    product : nisarqa.NonInsarProduct
        The input non-InSAR (RSLC, GSLC, GCOV) product.
    stats_h5 : h5py.File
        Handle to the output HDF5 file.
```

Reply:
Good catch! I went through all of the docstrings in `nisarqa/src/nisarqa/utils/stats_h5_writer/setup_writer.py` and updated them to be consistent with this comment.